WORKSPACE

```diff
@@ -22,11 +22,35 @@ docker_repositories()
 
 docker_pull(
     name = "distroless-base",
-    digest = "sha256:d19b95495f226aa64cd63a95863aca4da123bde22410a273e68e091113222e4f",  # latest
+    # latest circa 2017/10/06
+    digest = "sha256:66f5c4ecd54ee762e6e7329fcac11b3b6bfd6696044323ee8eabfee9601cc707",
     registry = "gcr.io",
     repository = "distroless/base",
 )
 
+load(
+    "@io_bazel_rules_docker//go:image.bzl",
+    _go_repositories="repositories",
+)
+
+_go_repositories()
+
+docker_pull(
+    name = "alpine-base",
+    # 0.1 as of 2017/10/06
+    digest = "sha256:c181891ef337fa5342597b92714e35bf381b15a2b05db655948310a44a3b4575",
+    registry = "gcr.io",
+    repository = "k8s-prow/alpine",
+)
+
+docker_pull(
+    name = "git-base",
```

Comment:
Can these two `docker_pull` rules be defined in the `prow/BUILD` file because they are Prow specific or is it required that they are defined in `WORKSPACE`?

Reply:
`docker_pull` is a repository rule, so it must be invoked during `WORKSPACE`.  We can pull it into a `.bzl` file under `prow/`, but we would still have to do something like this in `WORKSPACE`:
```python
load("//prow:foo.bzl", "prow_deps")

prow_deps()
```

where `prow/foo.bzl` is something like:
```python
load("@io_bazel_rules_docker//docker:docker.bzl", "docker_pull")

def prow_deps():
   docker_pull(...)
   docker_pull(...)
```

I'd probably recommend leaving this as-is and considering such a refactoring more globally, if you'd like to head that direction.  WDYT?

Reply:
Lets leave it as is.